demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -53,6 +55,7 @@ def is_valid_file(self, validate_rn=True):
             self.is_id_equals_name(),
             self.is_docker_image_valid(),
             self.is_valid_feed(),
+            self.is_valid_display_name(),
```

Comment:
decided on putting it in here and not in `is_backward_compatible`? what is the effort to make a pr to the content repo to fix that?

Reply:
opened a pr on this and will add it to backward_compatible